standards/certification/digisov-and-cert.md

```diff
@@ -0,0 +1,121 @@
+# Digital Sovereignty and SCS certification
+
+## The taxonomy of digital sovereignty
+
+As published in [DuD](https://rdcu.be/cWdBJ) (German, English version in
+[the cloud report](https://the-report.cloud/why-digital-sovereignty-is-more-than-mere-legal-compliance/))
+and being summarized nicely in a [cloudahead article](https://www.cloudahead.de/der-freiheitskampf-des-sovereign-cloud-stacks),
+we differentiate between several levels of digital sovereignty.
+We'll skip stage 0, introduced by Gregor Schuhmacher in his description, which
+specifies using a cloud at all as the pre-step to be taken. This has relevance,
+as some companies continue to call solutions that are not on-demand, not
+self-service API driven, not metered
+(see [NIST definition of cloud](https://nvlpubs.nist.gov/nistpubs/legacy/sp/nistspecialpublication800-145.pdf))
+to be (private) clouds. We talk about real clouds, where deployment of infrastructure
+is API-driven, unlocking DevOps teams productivity.
```

Comment:
I would suggest a different style here:

I think this is a very good place to define our taxonomy and reflect upon it.
I think it is also good to point to the DuD, cloudical and cloudahead articles - however I would not dive into "We'll skip stage 0, introduced by Gregor Schuhmacher in his description, which
 specifies using a cloud at all as the pre-step to be taken. This has relevance," because with that you mix the styles. While we're in the process of a (more or less) formal definition, you here fall into a conversational style.

Reply:
True, good observation. Will fix.